boot/zephyr/CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ zephyr_library_sources(
   ${BOOT_DIR}/bootutil/src/image_rsa.c
   ${BOOT_DIR}/bootutil/src/image_ec256.c
   ${BOOT_DIR}/bootutil/src/image_ed25519.c
+  ${BOOT_DIR}/bootutil/src/bootutil_misc.c
```

Comment:
This is not supposed to be part of this PR, is it? Maybe needs a rebase?

Reply:
rebased